baremetal/metal3machine_manager_test.go

```diff
@@ -565,6 +572,14 @@ var _ = Describe("Metal3Machine manager", func() {
 					ExpectedHostName: host2.Name,
 				},
 			),
+			Entry("Ignore hostWithUnhealthyAnnotation and pick host2, which lacks a ConsumerRer",
```

Comment:
consumerRef ??

Reply:
@kashifest this is testing the selection of the host. Unhealthy is ignored and one that does not yet have consumer is selected for machine.

Reply:
I think @kashifest was referring to a typo.

Reply:
I understand that. I meant it should be "ConsumerRef" not "ConsumerRer" :D